rpcserverhandlers_test.go

```diff
@@ -3488,6 +3488,80 @@ func TestHandleGetTxOutSetInfo(t *testing.T) {
 	}})
 }
 
+func TestHandleLiveTickets(t *testing.T) {
+	t.Parallel()
+
+	tkt1, err := chainhash.NewHashFromStr("1f6631957b4060d81ba7e760ec9c8150ba028eb051ddadf2b9749a5ccda1a955")
+	if err != nil {
+		t.Fatalf("[NewHashFromStr] unexpected error: %v", err)
+	}
+	tkt2, err := chainhash.NewHashFromStr("eca7e802590df60f7d300b6170f63dfab213b26421ed2e70de3ec2224cb9e460")
+	if err != nil {
+		t.Fatalf("[NewHashFromStr] unexpected error: %v", err)
+	}
+
+	testRPCServerHandler(t, []rpcTest{{
+		name:    "handleLiveTickets: no live tickets",
+		handler: handleLiveTickets,
+		cmd:     &types.LiveTicketsCmd{},
+		mockChain: func() *testRPCChain {
+			return defaultMockRPCChain()
+		}(),
+		result: types.LiveTicketsResult{
+			Tickets: []string{},
+		},
+	}, {
+		name:    "handleLiveTickets: two live tickets",
+		handler: handleLiveTickets,
+		cmd:     &types.LiveTicketsCmd{},
+		mockChain: func() *testRPCChain {
+			chain := defaultMockRPCChain()
+			chain.liveTickets = []chainhash.Hash{*tkt1, *tkt2}
+			return chain
+		}(),
+		result: types.LiveTicketsResult{
+			Tickets: []string{tkt1.String(), tkt2.String()},
+		},
+	}})
```

Comment:
Missing a test to hit the case where `s.cfg.Chain.LiveTickets()` returns an err (can see other test cases that mock an err for an example). And similarly for `s.cfg.Chain.MissedTickets()` in the tests for `TestHandleMissedTickets`